Library/Formula/hashcat.rb

```diff
@@ -0,0 +1,39 @@
+class Hashcat < Formula
+  desc "Advanced CPU-based password recovery utility"
+  homepage "https://hashcat.net/hashcat/"
+  url "https://codeload.github.com/hashcat/hashcat/tar.gz/2.00"
+  sha256 "6325e6d75a4df3485adec00f74e5887326809c15ed31bfe74a12b62943245444"
+
+  head do
+    url "https://github.com/hashcat/hashcat.git"
+  end
+
+  depends_on "lzip"
+  depends_on "gcc" # with multilib
+  depends_on "gmp"
+
+  fails_with :clang do
+    cause <<-EOS.undent
+      This formula does not compile with clang, use GCC.
+    EOS
+  end
+  fails_with :llvm do
+    cause <<-EOS.undent
+      Upstream depends on GCC with multilib.
```

Comment:
Compilation passes, program appears to run, but the upstream recommendation about multilib makes me wonder if it works.